apps/organisasjon-mottak-service/src/main/resources/application.yml

```diff
@@ -2,25 +2,29 @@ AAD_ISSUER_URI: https://login.microsoftonline.com/62366534-1ec3-4962-8869-9b5535
 
 spring:
   application:
-    name: organisasjon-mottak-service
+    name: testnav-organisasjon-mottak-service
     description: Testtjeneste for å vise hvilke tjenester som er tilgjenelig. Skal ikke brukes til noe annet enn for å test lokalt. For integrasjon bruk kafka.
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: ${AAD_ISSUER_URI}/v2.0
+          accepted-audience: ${azure.app.client.id}, api://${azure.app.client.id}
 
 consumers:
   jenkins-batch-status-service:
-    url: https://jenkins-batch-status-service.dev.intern.nav.no
+    url: http://testnav-jenkins-batch-status-service.dolly.svc.cluster.local
     namespace: dolly
-    name: jenkins-batch-status-service
-    cluster: dev-fss
+    name: testnav-jenkins-batch-status-service
+    cluster: dev-gcp
   jenkins:
-    url: http://testnorge-batch-adeo-proxy.dolly.svc.nais.local
+    url: https://testnorge-batch-adeo-proxy.dev-fss-pub.nais.io
     namespace: dolly
     name: testnorge-batch-adeo-proxy
     cluster: dev-fss
 
-kafka:
-  bootstrapservers: b27apvl00045.preprod.local:8443,b27apvl00046.preprod.local:8443,b27apvl00047.preprod.local:8443
-  schemaregistryservers: http://kafka-schema-registry.tpa.svc.nais.local:8081
-  groupid: organisajon-mottak-service-v1
+kafka.groupid: testnav-organisajon-mottak-service-v1
```

Comment:
replace **organisajon** -> **organisasjon** på de 3 kafka.groupid, hvis det ikke blir for mye knot :P